.github/ISSUE_TEMPLATE/new-feature.md

```diff
@@ -0,0 +1,24 @@
+---
+name: New feature
+about: Submit new InfluxData product features that need to be documented.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**PR:** _Provide PR URL(s) for this feature (if available)_
```

Comment:
```suggestion
##### Add pull request URL(s)
_Provide pull request URL(s) for this feature (if available)_
```

Reply:
Same comment as above. This is markdown that gets rendered after the issue is created. We don't need imperative headings in issues.